src/config.js

```diff
@@ -7,5 +7,9 @@ module.exports = {
   baseURL: process.env.baseURL || 'http://localhost:8080',
   jwtSigningKey: new Buffer(jwtSigningKeyB64, 'base64'),
   sqlite: process.env.sqlite || './data/podverseWeb.sqlite',
-  databaseName: 'podverse'
+  databaseName: 'podverse',
+
+  // TODO: change the auth0 values to podverse.fm's clientId and domain before deployment
+  auth0ClientId: 'tTGQrl5CenMDdpzcKNpmLIginyRgBJNN',
```

Comment:
I see. I was thinking we'd want something more like this:

````
auth0ClientId: process.env.auth0ClientId || 'whatever'
````

But I wasn't sure what we should use as the 'whatever'. I'll stick the test account Auth0 stuff in there for now (mitchd.auth0.com), and we can use the real stuff when deployed.

I hear ya, it would have been much, much easier to template with nunjucks. On the (self-centered) bright side, I'm glad I've got some Webpack experience finally, and I'd like to understand how to get the most out of it...